app/models/account.rb

```diff
@@ -44,6 +44,9 @@ def self.policy_class
   end
 
   private
+  def individual_acct?
```

Comment:
To me, it's an open question whether this is the way this should be handled. Because `IndividualAccount` overrides the `path` getter to use the opener's login, and because we essentially duplicate these validations on login for the user, _and_ because creating an individual account is effected by using `User#create_individual_account` which copies `login` into `Account.path` there is _currently_ little value in validating both.

On the other hand, things could drift in the future. Maybe writing custom validations using `self[:path]` is the way to go?

Reply:
Yes, there totally could. Requires some further thought 🤔

Reply:
Ok, wrote validations that work directly on the path attribute, and the API now returns a `409` with the failure message when those validations fail.